packages/nimble-components/src/rich-text-editor/template.ts

```diff
@@ -6,19 +6,28 @@ import { iconBoldBTag } from '../icons/bold-b';
 import { iconItalicITag } from '../icons/italic-i';
 import { iconListTag } from '../icons/list';
 import { iconNumberListTag } from '../icons/number-list';
+import { errorTextTemplate } from '../patterns/error/template';
+import { iconExclamationMarkTag } from '../icons/exclamation-mark';
 
 // prettier-ignore
 export const template = html<RichTextEditor>`
     <template>
         <div class="container">
             <section ${ref('editorContainer')} class="editor-container">
             </section>
-            <section class="footer-section" part="footer-section">
+            <${iconExclamationMarkTag}
+                severity="error"
+                class="error-icon"
+                style="--ni-private-rich-text-editor-scrollbar-width: ${x => x.scrollbarWidth}px;"
+                ?scrollbar-width-calculated="${x => x.scrollbarWidth >= 0}"
```

Comment:
Don't write to attributes that don't actually exist (the public api for icons do not define a scrollbarWidthCalculated property nor should they).

Instead follow the pattern of text-area and set a class when scrollbar width is defined:
https://github.com/ni/nimble/blob/db5ebd42c334810bb29be9c86cee763da487b39b/packages/nimble-components/src/text-area/template.ts#L61

Reply:
Properties and attributes should be defined by a component itself, the css classes should be owned by component users. The rich text editor is a user of those icon components